maintain.js

```diff
@@ -1,25 +1,345 @@
-/* Copyright © 2021 Ríona Stokes and other contributors, MIT License. */
-'use strict'
+module.exports = () => {
 
-const Fs = require('fs')
+  const Filehound = require('filehound')
+  const Path = require('path')
+  const Fs = require('fs')
+  const Hoek = require('@hapi/hoek')
+  const Marked = require('marked')
 
+  const checkList = require('./checks')
+  const checkOps = checkOperations()
 
-async function Maintain() {
-  console.log('MAINTAIN 0.0.2')
+  async function configDef(){
+    var argString = process.argv.slice(2)
```

Comment:
prefer let and const

Reply:
Resolved with commit 8ffd82fb6a786b28be7c2e95fd85e53de929fd9c